README.md

```diff
@@ -20,4 +21,5 @@ Generic
 
 Frameworks/Libraries
 
+- Reactjs `--template reactjs`
```

Comment:
Could you use just "react". because we like to go with "angular"

Reply:
reactjs renamed to react ;)